docs/src/troubleshooting.md

```diff
@@ -569,3 +569,29 @@ VOLNAME=$(kubectl get pv -o json | \
 
 kubectl delete pod/$PODNAME pvc/$PODNAME pvc/$PODNAME-wal pv/$VOLNAME
 ```
+
+### Error while bootstrapping data directory
+
+If you encounter the issue that your database cannot be initialized because it crashes during the bootstraping process, it is likely that your hugepage settings are set wrong.
+
+To check whether hugepages are enabled, run `cat /proc/meminfo` and check if huge pages are configured, their size and how many are free. If they are not enabled, you can adjust your cluster configuration as follows:
+```
+  postgresql:
+    parameters:
+      huge_pages: off
```

Comment:
I think the following should be suggested:
```suggestion
      huge_pages: "off"
```

because otherwise, this may cause the type-validation to fail:
```
Error from server: error when creating "posgres.yaml": admission webhook "mcluster.kb.io" denied the request: json: cannot unmarshal bool into Go struct field PostgresConfiguration.spec.postgresql.parameters of type string
```